src/daemon/bundle.rs

```diff
@@ -1,33 +1,84 @@
 // Copyright 2019-2023 ChainSafe Systems
 // SPDX-License-Identifier: Apache-2.0, MIT
 
-use crate::utils::db::{car_stream::CarHeader, car_util::load_car};
+use crate::{
+    networks::{generate_actor_bundle, ActorBundleInfo},
+    utils::db::car_util::load_car,
+};
 use anyhow::Context as _;
 use fvm_ipld_blockstore::Blockstore;
+use tracing::info;
 
-pub async fn load_actor_bundles(db: &impl Blockstore) -> anyhow::Result<CarHeader> {
-    const ERROR_MESSAGE: &str = "Actor bundles assets are not properly downloaded, make sure git-lfs is installed and run `git lfs pull` again. See <https://github.com/git-lfs/git-lfs/blob/main/INSTALLING.md>";
+/// Tries to load the actor bundle from the specified location to the blockstore. If the bundle is
+/// not present, it will be generated.
+pub async fn load_actor_bundles(db: &impl Blockstore) -> anyhow::Result<()> {
+    if !are_bundles_present(db, &crate::networks::ACTOR_BUNDLES) {
+        info!("Generating actor bundle");
+        let bundle = tempfile::Builder::new()
+            .prefix("forest-actor-bundle")
+            .suffix(".car.zst")
+            .tempfile()?;
+        generate_actor_bundle(bundle.path()).await?;
```

Comment:
Why do we still need bundling? How about downloading and loading only the missing ones?

Reply:
It was to avoid some code repetition, but I agree that it's wasteful. Let me make it right.

Reply:
@hanabi1224 fixed!

Reply:
Thanks!